requirements/detection.txt

```diff
@@ -3,3 +3,5 @@ mmcv-full==1.6.1
 #otxdet==0.3.2
 mmdet==2.25.1
 pytorchcv
+mmcls==0.23.2
+timm
```

Comment:
Don't we need to specify the version for stability?

Reply:
Good point, done